hack/test-local-registry.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/lib/init.sh"
+
+# Go to the top of the tree.
+cd "${OS_ROOT}"
+
+os::build::setup_env
+
+hack/build-go.sh cmd/dockerregistry
+dockerregistry="$( os::build::find-binary dockerregistry )"
+
+url="${DOCKER_REGISTRY_URL:-localhost:5000}"
+# find the first builder service account token
+token="$(oc get $(oc get secrets -o name | grep builder-token | head -n 1) --template '{{ .data.token }}' | base64 -D)"
+echo
+echo Login with:
+echo   docker login -p "${token}" -u user "${url}"
```

Comment:
This seems like indentation is intended. Without quoting the whole line, it won't be the case though.